includes/application/forwardApprover.php

```diff
@@ -63,20 +63,38 @@
 	if( isset( $_SESSION['PostName'] ) )
 	{
 		$UID = $_SESSION['PostName'];
+		if ( forwardApprover( $app_id ,$app_type, $UID ) == true )
+		{
+			echo "The application has been successfully FORWARDED to next Post<br>";
+		}
+		else
+		{
+			echo "Sorry, there was a problem updating<br>";
+			die();
+		}
 	}
 	else
 	{
 		$UID = $_SESSION['Username'];
-	}
-	
-	if ( forwardApprover( $app_id ,$app_type, $UID ) == true )
-	{
-		echo "The application has been successfully FORWARDED to next Post<br>";
-	}
-	else
-	{
-		echo "Sorry, there was a problem<br>";
-		die();
+		
+		$status = $_POST['status'];
+		
+		$User = findPending($status);
+		
+		if( $User == null )
+		{
+			echo "Error forwarding";
+			die();
+		}
+		else if ( forwardApprover( $app_id ,$app_type, $User ) == true )
+		{
+			echo "The application has been successfully FORWARDED to next Post<br>";
+		}
+		else
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after ELSE keyword; newline found

<!-- Comment by [SideCI](https://sideci.com) -->